lib/Model/Error.php

```diff
@@ -36,31 +44,49 @@ public static function fromException(Throwable $exception): Error
         );
     }
 
+    /**
+     * @return int
+     */
     public function getCode()
     {
         return $this->code;
     }
 
+    /**
+     * @return int
+     */
     public function getLine()
     {
         return $this->line;
     }
 
+    /**
+     * @return string
+     */
     public function getFile()
     {
         return $this->file;
     }
 
+    /**
+     * @return string
+     */
     public function getMessage()
     {
         return $this->message;
     }
 
+    /**
+     * @return string
+     */
     public function getClass()
     {
         return $this->class;
     }
 
+    /**
+     * @return string
+     */
     public function getTrace()
```

Comment:
ditto, seems these can all be real return types?

Reply:
B/C